bool is_reset

float32 delta_x
float32 delta_y
float32 delta_z
float32 delta_wrist

int8 gripper_state
---
bool result